src/convertToObject.js

```diff
@@ -1,18 +1,22 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const formatedStyles = {};
+
+  let propertyArray = sourceString.replace(/[\n]/gi, '').split(';');
```

Comment:
but let's not use `let` for arrays/objects, etc. Either create a copy with consts, or just chain methods like in [checklist](https://github.com/mate-academy/js_style-to-object/blob/master/checklist.md) 4th item